tests/test_scenes_api_action_field.py

```
@@ -0,0 +1,327 @@
+            "v4": {"action": "ignore"},
+        },
+    }
+
+    mock_request = AsyncMock()
+    mock_request.json = AsyncMock(return_value=request_data)
+
+    with patch("ledfx.api.scenes.save_config"):
+        response = await scenes_endpoint.post(mock_request)
+        data = json.loads(response.body.decode())
+
+        scene_id = data["scene"]["id"]
```

Ruff (0.14.5)

Local variable `scene_id` is assigned to but never used

Remove assignment to unused variable `scene_id`

(F841)